### PR TITLE
update arrange function

### PR DESCRIPTION
**Pull request by LilyTao0531**
Add code to arrange() function to enable sorting in ascending and descending orders.

### CODE REVIEW

File: src/tidyversetopandas/tidyversetopandas.py [hunk 1]
```diff
@@ -20,4 +20,5 @@ def arrange(df: pd.DataFrame, *col_name:str):
     >>> df = pd.DataFrame(np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=['a', 'b', 'c'])
     >>> df_sorted = tidyversetopandas.arrange(df, 'a', 'c')
     """
-    pass
\ No newline at end of file
+
+    return df.sort_values(by=list(col_name), ascending = ascending)
```
Comment (farrandi) [inline src/tidyversetopandas/tidyversetopandas.py:24]: nit: end in an extra line
```suggestion
    return df.sort_values(by=list(col_name), ascending = ascending)
    
```

File: src/tidyversetopandas/tidyversetopandas.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-def arrange(df: pd.DataFrame, *col_name:str):
+def arrange(df: pd.DataFrame, ascending: bool = True, *col_name:str):
     """
     Sort a Pandas dataframe in the ascending order
 
-    This function takes a Pandas dataframe and names of the columns, according to which the dataframe is sorted ascendingly
+    This function takes a Pandas dataframe and names of the columns, according to which the dataframe is sorted ascendingly/descendingly
 
     Parameters:
     -------
```
Comment (farrandi) [inline src/tidyversetopandas/tidyversetopandas.py:10]: Another nit: all 3 of our other function does not have the "---", so maybe remove it.

Also remove it under Returns
```suggestion
```

File: src/tidyversetopandas/tidyversetopandas.py [hunk 3]
```diff
@@ -20,4 +20,5 @@ def arrange(df: pd.DataFrame, *col_name:str):
     >>> df = pd.DataFrame(np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=['a', 'b', 'c'])
     >>> df_sorted = tidyversetopandas.arrange(df, 'a', 'c')
```
Comment (farrandi) [inline src/tidyversetopandas/tidyversetopandas.py:21]: Do not need to have tidyversetopandas for the example
```suggestion
    >>> df_sorted = arrange(df, 'a', 'c')
```

**Review by farrandi — CHANGES_REQUESTED** [review]
Looks good, gave some minor nits

File: src/tidyversetopandas/tidyversetopandas.py (hunk 3 above)
Comment (LilyTao0531) [reply src/tidyversetopandas/tidyversetopandas.py:21]: Good point. Thanks for checking!

File: src/tidyversetopandas/tidyversetopandas.py [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
     """
     Sort a Pandas dataframe in the ascending order
 
-    This function takes a Pandas dataframe and names of the columns, according to which the dataframe is sorted ascendingly
+    This function takes a Pandas dataframe and names of the columns, according to which the dataframe is sorted ascendingly/descendingly
 
     Parameters:
-    -------
     - df(pandas.DataFrame): The input dataframe object.
     - *col_name(string)
```
Comment (786213750) [inline src/tidyversetopandas/tidyversetopandas.py:12]: you can add description for the new flag
```suggestion
- ascending(bool): Specifies the sorting order. Set to True for ascending order, or False for descending order. Default is ascending order.
```

**Review by 786213750 — APPROVED** [review]
Everything looks great!

**Review by farrandi — APPROVED** [review]
LGTM

**Pull request merged**